gflownet.py

```diff
@@ -57,8 +57,14 @@ def __init__(self, config, logger, env, load_best_model=False):
 
         self.load_hyperparameters()
 
-        self.loginf = tf_list([1e6])
+        self.loginf = tf_list([1e3])
```

Comment:
why not putting 1e6 to be real big ?

Reply:
I was just matching with the previous code